WordPress/gutenberg
# [RNMobile] Fix inserter menu tabs rendering

`gutenberg-mobile` PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/3543





## Description


The inserter menu tabs were rendered by calling the functional component directly instead of rendering the component as JSX. Calling the functional components is discouraged because React doesn't render it as a component and therefore it leads to errors with hooks like `Rendered fewer/more hooks than expected`, [here is an interesting post about this issue](https://kentcdodds.com/blog/dont-call-a-react-function-component).

## How has this been tested?




**For testing this issue it's required to have reusable blocks (one would be enough) on the site used.**

1. Open the mobile app
2. Go offline by turning off the Wi-Fi connection and/or Mobile Data
3. Open a post page
4. Tap on ➕ button
5. Go online by turning on the Wi-Fi connection and/or Mobile Data
6. Observe that no error is displayed (red error screen) and that the app doesn't crash

This can be also tested by modifying the value of the prop `showReusableBlocks` to `false` passed in [this line](https://github.com/WordPress/gutenberg/blob/4edc47f5187e7f58dc1a9618ba8d732b5d072c00/packages/block-editor/src/components/inserter/menu.native.js#L211) while the inserter menu is open.

## Screenshots 

N/A

## Types of changes




Bug fix.

## Checklist:
